README.md

```diff
@@ -77,6 +77,8 @@ If you want to use `src` with your own Sourcegraph instance set the `SRC_ENDPOIN
 SRC_ENDPOINT=https://sourcegraph.example.com src search
 ```
 
+> NOTE: If on macOS, we recommend installing the `zip` and `diffutils` Homebrew packages as the related binaries on macOS are quite old. This will solve issues related to being unable to unzip archives and diffing files.
```

Comment:
```suggestion
> NOTE: If you want to use the `src actions exec` functionality on macOS, we recommend installing the `zip` and `diffutils` packages from Homebrew (or any other package manager that has up-to-date versions) as the versions that come with macOS are quite old. This will solve issues related to being unable to unzip archives and diffing files.
```